src/aiida/cmdline/commands/cmd_presto.py

```diff
@@ -92,13 +92,15 @@ def detect_postgres_config(
     except Exception as exception:
         raise ConnectionError(f'Unable to automatically create the PostgreSQL user and database: {exception}')
 
+    aiida_config_folder = AiiDAConfigDir.get_configuration_directory()
```

Comment:
Maybe it is better to just name the methods `AiiDAConfigDir.get()` and `AiiDAConfigDir.set()`?